src/topic/TopicMessageQuery.js

```diff
@@ -438,6 +438,13 @@ export default class TopicMessageQuery extends Query {
                     const message =
                         error instanceof Error ? error.message : error.details;
 
+                    if (this._handle?._unsubscribed) {
+                        console.log(
+                            "Subscription has been cancelled. Stopping retries.",
+                        );
```

Comment:
remove the console.log. We use console.logs only in our examples but not in the implementation itself. 
```suggestion
```